docs/react-hook-form.md

```diff
@@ -43,9 +43,9 @@ Using our schema is easy enough, using a resolver:
 import { zodResolver } from '@hookform/resolvers/zod';
 import { useForm } from 'react-hook-form';
 
-formDefaultValues = {
+const formDefaultValues : Partial<FormModel> = {
   name: '',
-  age: 0,
+  age: 0, // I am a bit concerned about this as Im not sure its a good idea to default form values to invalid values.
   idHobbies: [],
 };
```

Comment:
I would suggest collocating default values with the schema. Altenratively, we need to consider a way to have "dynamic" default values, for scenarios where we preload data from memory and such.

Reply:
In retrospect, deciding the default values is more of business decision than anything else. So you can ignore that concern.